assets/js/googlesitekit/widgets/datastore/areas.js

```diff
@@ -198,9 +199,12 @@ export const selectors = {
 
 		const { areas, contextAssignments } = state;
 
-		return Object.values( areas ).filter( ( area ) => {
-			return contextAssignments[ contextSlug ] && contextAssignments[ contextSlug ].includes( area.slug );
-		} ).sort( ( areaA, areaB ) => areaA.priority - areaB.priority );
+		return sortObjectMapByKey(
+			Object.values( areas ).filter( ( area ) => {
```

Comment:
See above, we're passing an array to the function which right now expects an object.